### DISCUSSION TITLE
Windows: capturing the output of the plotting process on the CLI

### DISCUSSION THREAD

**Discussion by subz390** [Chia-Network/chia-blockchain#2535] [Q&A]
I'm having unreasonable levels of resistance capturing the output of the plotting process in Windows.  I've been struggling for two days already and tried everything that people have suggested in my Discord.  These methods listed below work for me when used with other scripts and process, but not with Chia.exe.
1. Windows CMD - `command > filename` nothing is piped to the logfile.  https://ss64.com/nt/syntax-redirection.html
2. Windows PowerShell - `Start-Process` with `-RedirectStandardError` I get a partial amount of log at the top of the file and then nothing and `-RedirectStandardOutput` I get nothing at all in the log file, and all of it to the console. https://ss64.com/ps/start-process.html

So quite frankly I'm at a loss right now, what more can I do?  any suggestions please.

**Comment by djdookie** [comment]
I just suggested that the CLI should also write plotter logs autonomously without the need to manually capture CLI output:
https://github.com/Chia-Network/chia-blockchain/issues/3530
